client/src/banners/active-banner.tsx

```diff
@@ -87,24 +91,48 @@ function Banner(props: BannerProps) {
   );
 }
 
-function CommonSurveyBanner({ onDismissed }: { onDismissed: () => void }) {
+// function CommonSurveyBanner({ onDismissed }: { onDismissed: () => void }) {
+//   const ga = useGA();
+
+//   return (
+//     <Banner
+//       id={COMMON_SURVEY_ID}
+//       title={"Learning web development survey"}
+//       copy={
+//         "Help us understand how to make MDN better for beginners (5 minute survey)"
+//       }
+//       cta={"Take the survey"}
+//       url="https://www.surveygizmo.com/s3/6175365/59cfad9c04cf"
+//       newWindow
+//       onDismissed={onDismissed}
+//       onCTAClick={() => {
+//         ga("send", {
+//           hitType: "event",
+//           eventCategory: CATEGORY_LEARNING_SURVEY,
+//           eventAction: "CTA clicked",
+//           eventLabel: "banner",
+//         });
+//       }}
+//     />
+//   );
+// }
+function MDNPlusPlusBanner({ onDismissed }: { onDismissed: () => void }) {
```

Comment:
The git diff, here in the PR UI, is confusing. I think it's better to look at the file as in outside the diff.